### DISCUSSION TITLE
[Projects Beta] Create milestones and labels from Projects?

### DISCUSSION THREAD

**Discussion by creasty** [community/community#6040] [Projects and Issues]
Projects can gather issues from multiple repositories and centralize the management, so the experience of combining it with milestones and labels should be more streamlined.
Now in Beta, milestones and labels are displayed as columns and can be edited directly; Although they look like they're a part of the project's configuration, selectable values of the fields vary in which repository the issue is filed.

Technically it might not be a scope of Projects Beta per se, but nonetheless, I'd like to create milestones and labels from Projects, or automatically create/populate the dataset across specified repositories somehow.

For the time being, I'm thinking of setting up github workflow in our main repo and reflect a predefined list of milestones and labels to other repos. But I'd love to see it in the product.

**Comment by karlhorky** [comment]
Potentially related are project-wide (cross-repo) milestones, proposed here: https://github.com/github/feedback/discussions/6296
